ecommerce/frontend/src/config.js

```diff
@@ -1,5 +1,5 @@
 export const config = {
-    apiUrl: process.env.REACT_APP_API_URL ?? 'http://localhost:8000',
+    apiUrl: 'http://ec2-18-189-28-20.us-east-2.compute.amazonaws.com:8000', // process.env.REACT_APP_API_URL ?? 'http://localhost:8000',
 }
 
```

Comment:
do not change this otherwise won't work for different backends.

Reply:
Reverted back to original.